README.md

```diff
@@ -0,0 +1,5 @@
+# Delivery
+
+[![Build Status](https://travis-ci.org/AmazeeLabs/delivery.svg?branch=8.x-1.x)](https://travis-ci.org/AmazeeLabs/delivery) [![codecov](https://codecov.io/gh/AmazeeLabs/delivery/branch/8.x-1.x/graph/badge.svg)](https://codecov.io/gh/AmazeeLabs/delivery) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/AmazeeLabs/delivery/badges/quality-score.png?b=8.x-1.x)](https://scrutinizer-ci.com/g/AmazeeLabs/delivery/?branch=8.x-1.x)
+
+Readme coming soon.
```

Comment:
We could perhaps include a short description about the module. We have already done some text somewhere (I didn't see it in Silverback docs), I think @chindris or @monya007 might have done some

Reply:
Will add in a subsequent update.